src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -473,7 +473,7 @@ private void verifyEmail(EmailWrapper email, String recipient, String subject, S
         EmailChecker.verifyEmailContent(emailContent, emailContentFilePath);
 
         // check email body for no left placeholders
-        assertFalse(emailContent.contains("${"));
+        //assertFalse(emailContent.contains("${"));
```

Comment:
> I thought that this should not be the best approach

Why do you think it is not?

> so I commented the line to see what happens, and all the tests passed.

So there is a regression occurring, e.g. I suspect that the placeholder `${coOwnersEmails}` is not getting replaced. This is what you should fix instead of commenting out (essentially removing) tests.

Reply:
>Why do you think it is not?

I meant Commenting the test seemed like a bad approach to handle this error.

>So there is a regression occurring, e.g. I suspect that the placeholder ${coOwnersEmails} is not getting replaced. This is what you should fix instead of commenting out (essentially removing) tests.

Okay, I am working on this.

Reply:
**Update:** This bug got fixed after I changed the `userEmailFragment-sessionClosedAdditionalInstructionsFragment.html` file. All the `EmailGeneratorTest`s are passing now. <br>
Should I push the changes for a review?